corehq/apps/app_manager/id_strings.py

```diff
@@ -206,7 +206,7 @@ def module_locale(module):
 @pattern('forms.m%df%d')
 def form_locale(form):
     return "forms.m{module.id}f{form.id}".format(module=form.get_module(),
-                                                  form=form)
+                                                 form=form)
```

Comment:
nit: This looks short enough to fit on one line right?

Reply:
fixed ⭐️